cmd/vdev_id/vdev_id

```diff
@@ -565,7 +564,7 @@ fi
 TOPOLOGY=${TOPOLOGY:-sas_direct}
 
 # Should we create /dev/by-enclosure symlinks?
-if [ "$ENCLOSURE_MODE" = "yes" -a "$TOPOLOGY" = "sas_direct" ] ; then
+if [ "$ENCLOSURE_MODE" = "yes" ] && [ "$TOPOLOGY" = "sas_direct" ] ; then
```

Comment:
I would prefer then "&&" be inside the test rather than two separate tests, but I don't think it matters.